docs/book/src/developer/providers/migrations/v1.5-to-v1.6.md

```diff
@@ -31,3 +31,45 @@ maintainers of providers and consumers of our Go API.
 
 ### Suggested changes for providers
 
+- To enable clients to encode and decode your API, your types must be able to be registered within a  [scheme](https://pkg.go.dev/k8s.io/apimachinery/pkg/runtime#Scheme). The default kubebuilder scheme builder introduces a dependency on controller-runtime to your API types.
+
+  To mitigate it, Cluster API removed the reliance on controller-runtime scheme builder (see [#9045](https://github.com/kubernetes-sigs/cluster-api/pull/9045)) in order to reduce the number of dependancies being introcuded to its API types and make API types easily consumable by other projects.
```

Comment:
In order to reduce dependencies for API package consumers, CAPI has diverged from the default kubebuilder scheme builder. This new pattern may also be useful for reducing dependencies in provider API packages. For more information [see the implementers guide.](../implementers-guide/create_api#registering-apis-in-the-scheme)

Reply:
@killianmuldoon thank you, updated the PR and it is much cleaner now!